src/main.tsx

```diff
@@ -9,6 +9,10 @@ const router = createBrowserRouter([
     path: '/',
     element: <HomePage />,
   },
+  {
+    path: '/how-to-play',
```

Comment:
Can we name this route something else?

```suggestion
    path: '/how-to-play-this-game',
```